pallets/author-mapping/src/weights.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for pallet_author_mapping
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 3.0.0
+//! DATE: 2021-07-13, STEPS: `[32, ]`, REPEAT: 64, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/moonbeam
+// benchmark
+// --chain
+// dev
+// --execution=wasm
+// --wasm-execution=compiled
+// --pallet
+// pallet_author_mapping
+// --extrinsic
+// *
+// --steps
+// 32
+// --repeat
+// 64
+// --raw
+// --template=./benchmarking/frame-weight-template.hbs
+// --output
+// /tmp/
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{
+	traits::Get,
+	weights::{constants::RocksDbWeight, Weight},
+};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_author_mapping.
+pub trait WeightInfo {
+	fn add_association() -> Weight;
+	fn update_association() -> Weight;
+	fn clear_association() -> Weight;
+}
+
+/// Weights for pallet_author_mapping using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	fn add_association() -> Weight {
+		(74_659_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(6 as Weight))
```

Comment:
```suggestion
      // Manually added one read to account for read executed by ParachainStaking::is_candidate
			.saturating_add(T::DbWeight::get().reads(7 as Weight))
```
it's actually 7 reads because the benchmarking execution was not including the `ParachainStaking::is_candidate` read

Reply:
removed in #600 no need to manually edit the generated weights file anymore